App/Sagas/Migration.ts

```diff
@@ -255,12 +253,13 @@ export function * runRecurringMigrationTasks () {
 }
 
 // Will error for any non-success
-async function announceId(currentPeerId: string, previousPeerId: string, currentAddress: string, previousUsername?: string) {
+async function announcePeerDetails(peerDetails: PeerDetails) {
   const headers = {'Content-Type': 'application/json'}
-  const payload: { [key: string]: string } = { peerId: currentPeerId, previousId: previousPeerId, address: currentAddress }
-  const username = previousUsername ? previousUsername.trim() : ''
-  if (username.length > 0) {
-    payload['username'] = username
+  const contactInfoJsonString = JSON.stringify(peerDetails.currentContactInfo)
```

Comment:
Sending a JSON string of the `ContactInfo` so we can store it as a string in the database